.intellijPlatform/localPlatformArtifacts/bundledPlugin-org.jetbrains.plugins.yaml-2024.2.0.2+665.xml

```diff
@@ -0,0 +1,10 @@
+<ivy-module version="2.0">
```

Comment:
Are we going to need a separate xml file per JetBrains release version?

Reply:
Nope, I'll delete it